### ISSUE TITLE
For Polygon NFT gating, add some text...

### ISSUE THREAD

**Issue by mrose17** [brave/brave-talk#1127]
please update "Currently supports Ethereum ERC-721 NFTs" to "Currently supports ERC-721 and ERC-1155". Note that when you re-sync the feature branch with `main`, the text will also include SOL NFTs... So, you may just want to change it now to
```
Currently supports ERC-721 and ERC-1155 NFTs and Solana Metaplex NFTs
```
Also, for the tooltip that pops up when someone mouses over an avatar, in addition to showing the title, I suggest appending " (network)" where you get network by looking at the `nft_id` and taking the text upto, but not including, the period. (If there's no period, then to be safe, don't append anything.)

**Comment by mrose17** [comment]
@davidsuh11 - please include this functionality into the feature branch that addresses #1072

**Comment by mrose17** [comment]
Fixed by #1182
